packages/menu-bar/src/vaadin-menu-bar-interactions-mixin.js

```diff
@@ -455,4 +464,40 @@ export const InteractionsMixin = (superClass) =>
         this._subMenu.close();
       }
     }
+    
+     /** @private */
+    _requestClose() {
+      this._preventClose = false;
+      // wait if something has to prevent the close event
+      setTimeout(() => {
+        if (!this._preventClose) {
+          this._close(false);
+        }
+      }, 300);
```

Comment:
We should avoid hardcoding this number to make it configurable similar to how `vaadin-tooltip` does it:

- property / attribute - e.g. `<vaadin-tooltip hide-delay="300">`
- global static method - e.g. `Tooltip.setDefaultHideDelay(300)`

As we are talking about a new feature, we might want to chance to come up with a proper name for it.

@jouni @rolfsmeds What do you think about this?